### DISCUSSION TITLE
[Feature]: Analogue Document->getElement in twig

### DISCUSSION THREAD

**Discussion by esisl** [pimcore/pimcore#17577] [General]
### Feature description

In version 6, it was possible to access a document from a template

```
$doc = Document::getById($id);
echo $doc->getElement('name');

```
Is there an analogue in version 11 for twig templates?
I don't understand how to use pimcore_document for this

**Comment by kingjia90** [comment]
Thank you for your prompt!

This should work, not familiar with Pimcore 6, but getElement is only present in the `link` document type so far, so the code should look like
```
    {% set document = pimcore_document(236) %}
    {% if document is pimcore_document_link %}
        {{ document.getElement() }}
    {% endif %}
```
What would/should `getElement('name')` return?

Moving to discussion area, as it doesn't sound like a Feature

**Comment by esisl** [comment]
Using an anti-scientific poke, I found that it is possible like this:
doc.editable('field_name').data()

But there's another problem. The template returns a string recycled into htmlspecialchars even if I try to force wrap it with htmlspecialchars_decode in TwigFunction

How do I get a string WITHOUT forced transcoding?

**Comment by esisl** [comment]
I'm sorry, I found it.

It turns out that it is necessary to do
{{ doc.editable('field_name').data()|raw }}

It seems very strange to me, but it's good that at least it is...
